BUILDING.md

```diff
@@ -251,7 +251,7 @@ To test if Node.js was built correctly:
 > Release\node -e "console.log('Hello from Node.js', process.version)"
 ```
 
-### Android/Android-based devices (e.g. Firefox OS)
+### Android/Android-based devices (for example: Firefox OS)
```

Comment:
This would probably be better/clearer as `(Firefox OS, etc.)` rather than `(for example: Firefox OS)`.